README.md

```diff
@@ -1,2 +1,106 @@
 # in-toto
-in-toto is a series of scripts to protect the supply chain integrity.
+Protecting the software supply chain integrity
+
+in-toto guarantees that the end-user (or client) is able to verify that the entire development life cycle has been conducted as per the specified layout and that each of the functionaries (eg. developers) have performed the specified tasks and there haven't been any malicious changes in the files.
```

Comment:
I don't know if we should assume that the term "layout" is self-explanatory. What do you guys think?